client/tests/jest/jest.setup.js

```diff
@@ -4,10 +4,25 @@ import "fake-indexeddb/auto";
 
 import Vue from "vue";
 
+// not available in jsdom, mock it out
+Element.prototype.scrollIntoView = jest.fn();
+
 // Set Vue to suppress production / devtools / etc. warnings
 Vue.config.productionTip = false;
 Vue.config.devtools = false;
 
+// if we do jest.mock(module) where modules imports Vue, it will create a clean Vue context
+// that ignores the above tips. So pre-emptively mock out Vue in these contexts with the following
+// block of code. GridList.test.js is a good example of this, when ran (yarn test GridList.test.js)
+// it will bypass the attempt to set the Vue config values above and print the messages anyway without
+// this block.
+jest.mock("vue", () => {
+    const Vue = jest.requireActual("vue"); // Import the actual Vue instance
+    Vue.config.productionTip = false; // Disable production tip
+    Vue.config.devtools = false; // Disable production tip
```

Comment:
```suggestion
    Vue.config.devtools = false; // Disable vue devtools
```

Reply:
Thanks!

Reply:
Thank you!  This is fantastic cleanup for the tests.